src/Common/src/System/Net/Security/NegotiateStreamPal.Unix.cs

```diff
@@ -293,7 +358,61 @@ internal static string QueryContextAuthenticationPackage(SafeDeleteContext secur
             ref byte[] resultBlob,
             ref ContextFlagsPal contextFlags)
         {
-            throw new PlatformNotSupportedException(SR.net_nego_server_not_supported);
```

Comment:
Can we delete the associated string from the .resx, or is it still needed?

Reply:
I think it can be deleted.

Reply:
It's still referenced in a few places.

Reply:
I looked at the places it's used.  We can leave the string in the .resx for now.  But I think we should be able to eventually fix the other PlatformNotSupportedException's later and remove the string.